etc/vagrant/install.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# Script to set up a Django project on Vagrant.
+PROJECT_NAME=$1
+
+DB_NAME=$PROJECT_NAME
+DB_USERNAME=$PROJECT_NAME
+DB_PASSWD=$PROJECT_NAME
+
+PROJECT_DIR=/home/vagrant/$PROJECT_NAME
```

Comment:
do we need to install it in some subdirectory?

if we will install it directly in home directory of `vagrant` user i will allow to avoid one step from `Vagrant setup` manual (`cd volontulo`)